src/main/kotlin/it/pagopa/ecommerce/helpdesk/exceptions/InvalidSearchCriteriaException.kt

```diff
@@ -14,6 +13,6 @@ class InvalidSearchCriteriaException(reason: String) : ApiError(reason) {
 
     constructor(
         searchCriteriaType: String,
-        productDto: ProductDto
+        productDto: String
```

Comment:
maybe v2 for this exceptino?

Reply:
Changed the behavior of the InvalidSearchCriteriaException class here: d23ddcdfe6c625617887c40f96d5489afc749c74

Reply:
sounds good!